aries-site/src/pages/_app.js

```diff
@@ -59,15 +60,159 @@ const backgroundImages = {
  * The `Component` prop is the active `page`, so whenever you
  * navigate between routes, `Component` will change to the new `page`.
  */
+
+export const ViewContext = createContext(undefined);
+
 function App({ Component, pageProps, router }) {
   const route = router.route.split('/');
 
-  // necessary to ensure SkipLinks can receive first tab focus
-  // after a route change
+  //state that holds the update information within the last 30 days
+  const [wholeViewHistory, setWholeViewHistory] = useState({});
```

Comment:
```suggestion
  const [contentHistory, setContentHistory] = useState({});
```
small wording suggestion